CONTRIBUTING.md

```diff
@@ -720,6 +720,10 @@ And for a release build of Tachiyomi:
 
 ### Android Debugger
 
+> [!IMPORTANT]
+> If you didn't build Tachiyomi from source with debug enabled and are using a release/beta APK, you **need** a rooted device.
```

Comment:
```suggestion
> If you didn't build Mihon from source with debug enabled and are using a release/beta APK, you **need** a rooted device.
```

Reply:
yeah it is kinda outdated in some places too... you can refer to "the app" instead of name droping

Reply:
I'm just gonna call it "the main app"